content/docs/start/metrics-parameters-plots.md

```diff
@@ -5,18 +5,19 @@ title: 'Get Started: Metrics, Parameters, and Plots'
 # Get Started: Metrics, Parameters, and Plots
 
 DVC makes it easy to track [metrics](/doc/command-reference/metrics), update
-[parameters](/doc/command-reference/params), and visualize performance with
-[plots](/doc/command-reference/plots). These concepts are introduced below, and
-[Experiments](/doc/start/experiments) shows how to combine them to run and
-compare many iterations of your ML project.
+<abbr>parameters</abbr>, and visualize performance with
+[plots](/doc/command-reference/plots). These concepts are introduced below.
```

Comment:
Not sure about removing the Experiments link (recently introduced by @dberenbaum I think).

Reply:
experiments link (well, tooltip) added to an aside/note in the next para.

Reply:
The rationale for including the link was to encourage users to look at experiments for how to use these concepts, since tracking parameters and metrics without doing multiple experiments probably doesn't have much value. What's the rationale for removing it? Then we can decide what makes sense.

Reply:
Hmm. This was my logic:

- page title: X Y Z
- first para: introduce X Y Z
- aside: see also A

whereas before it was:

- page title: X Y Z
- first para: X Y A(?) Z

Just to clarify @dberenbaum it's not *re*moved - simply moved:

![image](https://user-images.githubusercontent.com/10780059/117354021-c521d680-aea8-11eb-82ad-5c8c5739083c.png)

Reply:
Ah I didn't notice that was turned into a note either, sorry my bad. Looks OK to me then! Except that it's not clear what "Read on to see how it's done" refers to, maybe should be part of the top p? But that p already has "These concepts are introduce below," should prob be either or